fix(types): Update polkadot-js deps to get the latest types (#156)
paritytech/txwrapper-core: README.md

1. Ensure that your version of `npm` is 7 or above. Check with `npm --version` and, if needed, upgrade with `npm install -g npm` (which may need to be prefixed with `sudo` depending on the permissions set on your global `node_modules` folder). **If this is not true, an empty binary will be pushed on publish.**

2. Checkout a branch `name-update-deps`.

3. Make sure to update the resolutions inside of the `package.json` to match polkadot-js [here](https://github.com/polkadot-js/apps/blob/master/package.json).

>     **note**: Remove any carets ("^") from the resolution versions. This is just for extra safety, and removes any concern around SemVer issues.

Reviewer: does this apply for just `dependencies` (i.e, not dev-dependencies)?

I got confused see comment below

Reviewer 2: It applies for all dependencies.

Author: Ah maybe my text wasn't clear enough then! I meant specifically from the "resolutions", but now I read it back that isn't clear. I'll tweak the words :)

Reviewer: it's quite clear but given that I'm such `typescript noob` I didn't get that you meant in the  "resolutions" field.

Author: It's hopefully super clear now; ideally you don't need to know any TS to be able to follow, so it was good to get your feedback :D

4. Ensure we have the latest polkadot-js dependencies by running the command below. If all packages are already up to date you may skip to the "Publishing" section below.
    **Note:** what follows assumes `yarn` at version 2.4.2 or above.

    ```bash
    yarn up "@polkadot/*"
    yarn up "@polkadot/apps-config@beta"
    yarn
    ```

> 5. Ensure there are no issues by running the following commands. If any type errors occur due to the updated dependencies, you may file an issue [here](https://github.com/paritytech/txwrapper-core/issues).

Reviewer 3: ```suggestion
5. Ensure there are no issues by running the following commands. If any type errors occur due to the updated dependencies, please file an issue [here](https://github.com/paritytech/txwrapper-core/issues).
```
